spec/features/questions_feature_spec.rb

```diff
@@ -53,6 +54,13 @@
     expect(page).to have_content "1 error prohibited this question from being saved:"
   end
 
+  it 'disallows non owners to make event questions' do
+    click_on "Sign out"
+    sign_in_as(user_two)
+    visit event_path(event)
```

Comment:
It works, as in the test passes, but visiting event_path(event) should be an Event Features test, testing that a user can't visit someone else's event. This test doesn't test that a user can't create a question on someone else's event.

To test that a user can't create a question on someone else's event, you need to test that they can't visit `new_event_question_path(event)` where `event` is someone else's event.